source/includes/api/core/_user_groups.md

```diff
@@ -20,18 +20,18 @@ Content-Type: application/json
 
 [
   {
-    "id" : "1",
-    "name" : "dc comics",
-    "parent_id" :  null,
-    "default" : "false",
-    "position" : "1"
+    "id": "1",
+    "name": "dc comics",
+    "parent_id":  null,
+    "default": "false",
+    "position": "1"
   },
   {
-    "id" : "2",
-    "name" : "justice league",
-    "parent_id" : "1",
-    "default" : "true",
-    "position" : "1"
+    "id": "2",
+    "name": "justice league",
+    "parent_id": "1",
+    "default": "true",
+    "position": "1"
   }
```

Comment:
can we have consistency please. json should have a space before and after the `:`

```json
{
  "name" : "dc comics"
}
```